pkg/util/helm/convert.go

```diff
@@ -20,13 +22,33 @@ import (
 // MinHelmVersion is the minimum KUDO version that supports helm templating
 const MinHelmVersion = "0.2.0"
 
-func loadParameters(folder string) ([]kudo.Parameter, error) {
+func LoadChart(chartFolder string) (*HelmChart, error) {
```

Comment:
we can test this one separately as well if we want to. It just reads helm chart from filesystem in the expected format of the chart and outputs the chart